wordpress-mobile/WordPress-Android
# Fix out-dated locale used in new site creation flow

Fixes #9245 

When the user changes language in the app settings, the New Site Creation flow starts sending the new locale to the api.

To test:
1. Start Stetho - network tab
2. Go to App settings
3. Change language
4. My Site
5. Click on the + sign icon
6. Create WordPress.com site
7. Look into Stetho and notice the correct locale is being sent as query parameter

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
